ReleaseTooling/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -671,7 +671,18 @@ struct FrameworkBuilder {
         fatalError("Could not copy fat binary to framework directory for \(framework): \(error)")
       }
       do {
-        try fileManager.copyItem(at: infoPlist, to: infoPlistDestination)
+        var plistDictionary = try PropertyListSerialization.propertyList(
```

Comment:
Please add a comment with a reference to the bug so future us can find the context.

Reply:
> ... and confirm we don't do something strange on catalyst and macos like writing a one-line plist file.

Now that you mention it, I'm noticing that no plist is generated for macOS/Catalyst. That looks to be expected, correct?

Reply:
Correct